lang/node-javascript-obfuscator/Makefile

```diff
@@ -0,0 +1,85 @@
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NPM_NAME:=javascript-obfuscator
+PKG_NAME:=node-$(PKG_NPM_NAME)
+PKG_VERSION:=0.18.1
+PKG_RELEASE:=3
+
+PKG_SOURCE:=$(PKG_NPM_NAME)-$(PKG_VERSION).tgz
+PKG_SOURCE_URL:=http://registry.npmjs.org/$(PKG_NPM_NAME)/-/
```

Comment:
use https

Reply:
I need to move the resource to git (). The existing repository does not allow https.

Reply:
Changed